packages/augur-ui/src/modules/portfolio/components/portfolio-view/my-bets.tsx

```diff
@@ -73,35 +110,69 @@ export const MyBets = () => {
             <SquareDropdown
               options={MY_BETS_VIEW_BY}
               defaultValue={MY_BETS_VIEW_BY[0].value}
-              onChange={selected => setState({ ...state, viewBy: selected })}
-              minimalStyle
-            />
-          </span>
-          <span>
-            Market Status:{' '}
-            <SquareDropdown
-              options={MY_BETS_MARKET_STATUS}
-              defaultValue={MY_BETS_MARKET_STATUS[0].value}
               onChange={selected =>
-                setState({ ...state, marketStatus: selected })
+                setState({
+                  ...state,
+                  viewBy: selected,
+                  rows:
+                    MY_BETS_VIEW_BY[selected].label === EVENT
+                      ? MOCK_GAMES_DATA
+                      : MOCK_OUTCOMES_DATA,
+                })
               }
               minimalStyle
             />
           </span>
+          {showEvents ? (
+            <span>
+              Market Status:{' '}
+              <SquareDropdown
+                options={MY_BETS_MARKET_STATUS}
+                defaultValue={MY_BETS_MARKET_STATUS[0].value}
+                onChange={selected =>
+                  setState({ ...state, marketStatus: selected })
+                }
+                minimalStyle
+              />
+            </span>
+          ) : (
+            <span>
+              Bet Date:{' '}
+              <SquareDropdown
+                options={MY_BETS_BET_DATE}
+                defaultValue={MY_BETS_BET_DATE[0].value}
+                onChange={selected => setState({ ...state, betDate: selected })}
+                minimalStyle
+              />
+            </span>
+          )}
         </div>
-        <PillSelection
-          options={SPORTS_MARKET_TYPES}
-          defaultSelection={selectedMarketCardType}
-          large
-          onChange={selected =>
-            setState({ ...state, selectedMarketCardType: selected })
-          }
-        />
+        {showEvents && (
```

Comment:
I would use one pill selector here for this and the bottom one, and instead change it to `options={showEvents ? SPORTS_MARKET_TYPES : MARKET_STATE_TYPES}`